types/office-js-preview/index.d.ts

```diff
@@ -101578,6 +102596,14 @@ declare namespace Word {
              * [Api set: WordApi 1.1]
              */
             doubleStrikeThrough?: boolean;
+            /**
+             * Specifies a value that indicates whether the font has a hidden. True if the font is formatted as hidden text, otherwise, false.
```

Comment:
```suggestion
             * Specifies a value that indicates whether the font is hidden. True if the font is formatted as hidden text, otherwise, false.
```

Reply:
Updated